vmware_exporter/vmware_exporter.py

```diff
@@ -662,79 +674,83 @@ def _vmware_get_vms(self, metrics):
                         snapshot['timestamp_seconds'],
                     )
 
-        log("Finished vm metrics collection")
+        logging.info("Finished vm metrics collection")
 
     @defer.inlineCallbacks
     def _vmware_get_hosts(self, host_metrics):
         """
         Get Host (ESXi) information
         """
-        log("Starting host metrics collection")
+        logging.info("Starting host metrics collection")
 
         results, host_labels = yield parallelize(self.host_system_inventory, self.host_labels)
 
         for host_id, host in results.items():
-            labels = host_labels[host_id]
+            try:
```

Comment:
As above

Reply:
Implemented in the most up to date version